docker-compose.yml

```diff
@@ -36,9 +36,23 @@ services:
       - $MATCHING_SVC_PORT:$MATCHING_SVC_PORT
     environment:
       - PORT=$MATCHING_SVC_PORT
+      - REDIS_HOST=redis
+      - REDIS_PORT=$MATCHING_SVC_REDIS_PORT
+    depends_on:
+      - redis
 
   redis:
     image: redis:7.4-alpine
     restart: always
     ports:
       - 6379:6379
```

Comment:
Might be best to use `$MATCHING_SVC_REDIS_PORT:$MATCHING_SVC_REDIS_PORT` here, so that if we were to use redis elsewhere, we won't have any issues, though we need to find a way to have redisinsight not assume matching service's redis is at port 6379.